src/data/trainer-config.ts

```diff
@@ -259,6 +265,8 @@ export class TrainerConfig {
   public hasGenders: boolean = false;
   public hasDouble: boolean = false;
   public hasCharSprite: boolean = false;
+  public spriteNameLeft: string;
+  public spriteNameRight: string;
```

Comment:
```suggestion
  public spriteNameLeft?: string;
  public spriteNameRight?: string;
```
If not all trainers have them, better mark as optional to enforce checks for that in the rest of the code
Comment would also be good to explain in which circumstances this is used, and for what.

Also maybe `spriteKeyLeft` and `spriteKeyRight` might be better names? But I don't feel too strongly about that